c2corg_ui/static/menu-discourse/menu.html

```diff
@@ -0,0 +1,117 @@
+    </a>
+    <div class="menu-open-btn">
+      <i class="glyphicon glyphicon-align-justify"></i>
+    </div>
+  </div>
+  <ul class="nav">
+    <li class="main-menu">
+      <a href="//www.demov6.camptocamp.org">
+        <i class="glyphicon glyphicon-home"></i>
+        <span class="lang en menu-text">Home</span>
+        <span class="lang fr menu-text">Accueil</span>
+      </a>
+    </li>
+    <li class="main-menu">
+      <a href="//www.demov6.camptocamp.org/routes">
+        <i class="glyphicon glyphicon-arrow-right"></i>
+        <span class="lang en menu-text">Guidebook</span>
+        <span class="lang fr menu-text">Topoguide</span>
+      </a>
+    </li>
+    <li class="main-menu menu-selected">
+      <a href="//forum.demov6.camptocamp.org/">
+        <i class="glyphicon glyphicon-th-list"></i>
+        <span class="menu-text lang en">Forum</span>
+        <span class="menu-text lang fr">Forum</span>
+      </a>
+    </li>
+    <li class="main-menu">
+      <a href="//demov6.camptocamp.org/articles/697210">
+        <i class="glyphicon glyphicon-warning-sign"></i>
+        <span class="menu-text lang en">Accident database</span>
+        <span class="menu-text lang fr">Base de données accidents</span>
+      </a>
+    </li>
+    <li class="main-menu">
+      <a href="//demov6.camptocamp.org/articles">
+        <i class="glyphicon glyphicon-info-sign"></i>
+        <span class="menu-text lang en">Articles</span>
+        <span class="menu-text lang fr">Articles</span>
+      </a>
+    </li>
+    <li class="main-menu">
+      <a href="//demov6.camptocamp.org/articles/106726">
+        <i class="glyphicon glyphicon-comment"></i>
+        <span class="menu-text lang en">Association</span>
+        <span class="menu-text lang fr">Association</span>
+      </a>
+    </li>
+  </ul>
+
+  <footer>
+
+
+    <ul class="footer-nav">
+      <li><a href="//www.demov6.camptocamp.org/articles/106727/en/contact-us">contact</a></li>
+      <span class="bullet">&bull; </span>
+      <li>
+        <a href="//www.demov6.camptocamp.org/articles/121965">
+          <span class="lang en">Credits</span>
+          <span class="lang fr">Crédits</span>
+        </a></li>
+      <span class="bullet">&bull; </span>
+      <li><a href="//www.demov6.camptocamp.org/articles/106731">
+          <span class="lang en">Terms of use</span>
+          <span class="lang fr">Conditions d'utilisation</span>
+        </a></li>
+      <span class="bullet">&bull; </span>
+      <li>
+        <a href="//www.demov6.camptocamp.org/articles/106728">
+          <span class="lang en">Content license</span>
+          <span class="lang fr">Licences de contenu</span>
+        </a>
+      </li>
+    </ul>
+
+    <div class="social-networks">
+      <div class="icons">
+        <a href="https://twitter.com/camptocamporg"><img src="https://c866088.ssl.cf3.rackcdn.com/assets/twiiter40x40.png"></a>
```

Comment:
If i use the static images, for ex `...static/4c851e7/img/facebook.svg` will the middle `4c851e7` not change often? I mean, if it does change it will not impact the displaying of the image?